src/main/java/ru/itmo/sd/deadliner2bot/model/Todo.java

```diff
@@ -1,6 +1,13 @@
 package ru.itmo.sd.deadliner2bot.model;
 
-import javax.persistence.*;
+import javax.persistence.Column;
```

Comment:
Actually, `import` changes belong to previous commit (i'm not insist to do smth with it, but let you know)

Reply:
My IDE is configured to avoid wildcard imports, that's why this was affected.
It is a bit better practice in general to avoid naming conflicts, but I don't think this worth discussion. Will rollback to defaults.